haproxy/setup.py

```diff
@@ -16,7 +16,7 @@
     long_description = f.read()
 
 
-CHECKS_BASE_REQ = 'datadog_checks_base'
+CHECKS_BASE_REQ = 'datadog_checks_base>=11.0.0'
```

Comment:
Curious. Why 11.0.0?

Reply:
```
## 11.0.0 / 2020-02-22

...
* [Added] Add new deprecation. See [#5539](https://github.com/DataDog/integrations-core/pull/5539).
```

Reply:
The underlying agent changes are going out in this release